### DISCUSSION TITLE
Emoji Support | FixIt

### DISCUSSION THREAD

**Discussion by giscus** [hugo-fixit/FixIt#140] [General]
# Emoji Support | FixIt

Guide to emoji usage in Hugo

https://fixit.lruihao.cn/guides/emoji-support/

**Comment by scmiot** [comment]
这个测试一下

**Reply by Lruihao** [reply]
测试请用自己的仓库的 key 哦~

**Comment by scmiot** [comment]
已经改回来了，感谢



发自我的iPhone


------------------ Original ------------------
From: Cell ***@***.***&gt;
Date: Wed,May 25,2022 8:11 PM
To: Lruihao/FixIt ***@***.***&gt;
Cc: scmiot ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [Lruihao/FixIt] Emoji Support | FixIt (Discussion #140)





 
测试请用自己的仓库的 key 哦~

**Comment by asaraog** [comment]
टेस्ट
